jsonschema-core.xml

```diff
@@ -640,12 +638,12 @@
                     with an instance document.  The outermost dynamic scope is the
                     root schema of the schema document in which processing begins.
                     The path from this root schema to any particular keyword (that
-                    includes any "$ref" and "$recursiveRef" keywords that may have
+                    includes any "$ref" and "$dyanmicRef" keywords that may have
```

Comment:
typo: "dyanmicRef" -- this occurs several times throughout the document (also "dyanmicAnchor")

Reply:
@karenetheridge thanks- I must have messed up one of the merge conflict resolutions

Reply:
Fixed